dkms_autoinstaller

```diff
@@ -82,6 +82,6 @@ case "$1" in
 		# Ignore others on debian/ubuntu too
         ;;
     *)
-        echo $"Usage: $0 {start}"
+        echo "Usage: $0 {start}"
```

Comment:
This commit removes bashism, where the commit says "... is not a bash script". While there is another one aka the string manipulation `prog=${exec##*/}` further up.

I don't have a strong opinion if we either a) land only this change b) also address the `prog` line or c) change the shbang to bash. As long as the commit message reflects what's being done.

Reply:
`${var#...}` and `${var%...}` and the `##` and `%%` variants are valid Posix sh string manipulations, no bashisms.